docs/conf/portland/2018/cfp.rst

```diff
@@ -0,0 +1,105 @@
+:template: 2018/base.html
+
+Call for proposals
+==================
+
+Hello hello, fellow documentarians! It's that time of year again: We’re very
+excited to announce that we are now accepting talk proposals for our next North
+American conference, coming up on **May 6-8, 2018, in Portland, Oregon**.
+
+Every year, Write the Docs invites people from all across our community to come
+up on stage to share their insights and experience. Whether you’ve been writing
+documentation for decades or you’ve just started this year, we want to hear from
+you! Read on to learn more about the goals of the conference and what we look
+for in talk proposals.
+
+In the meantime, mark your calendars:
+
+**The deadline for submitting proposals is Midnight PST on January 08, 2018.**
+
+We'll let you know if your proposal has been accepted by late January.
+
+Conference goals
+----------------
+
+Write the Docs conferences are our chance to get together in-person to explore and celebrate the craft of documentation in a positive, inclusive environment.
+
+Between the talks on stage, the discussions in the unconf, and the conversations in the hallway, 100% of the content for our conferences comes from our community! Whether your job title is writer, developer, product manager, support advocate, librarian, or one of a hundred others, your perspective on what makes good docs is what this conference is all about.
+
+Our goal is to give documentarians a chance to connect and learn from each other. You'll have the chance to compare notes on what's happening in the industry, dig into questions of convention and good practice, and generally nerd out about all the things we love about documentation.
```

Comment:
maybe link documentarian to info page, and possibly sign-post it more for newcomers?

Reply:
convention > conventions?
practice > practices?

Reply:
if it's 'question of' they need to be singular, but we can recast if it bugs you